src/sensorthings.js

```diff
@@ -22,9 +27,9 @@ module.exports = (config) => {
   let router = express.Router();
   router.use('/', baseRouter);
 
-  resourceEndpoints.forEach(endpoint => {
-    router.use(route.generate(endpoint),
-               require('./routes/' + Case.snake(endpoint)));
+  entities.forEach(endpoint => {
+    router.use(route.generate(endpoint), associations,
```

Comment:
Ah! Do we need to attach the middleware to every route? I think if we attach it to route.generate() is enough, isn't it?